libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -130,6 +154,22 @@ assembly::Statement Parser::parseStatement()
 	return statement;
 }
 
+assembly::Case Parser::parseCase(bool _defaultCase)
+{
+	assembly::Case _case = createWithLocation<assembly::Case>();
+	if (_defaultCase)
+		expectToken(Token::Default);
```

Comment:
I think it is cleaner to remove the function argument and just decide based on the current token.